### DISCUSSION TITLE
怎么编译出能用uboot刷入的bin格式的镜像

### DISCUSSION THREAD

**Discussion by igmkgu** [immortalwrt/immortalwrt#1344] [Q&A]
红米ax6000 ，immortalwrt编译好的固件是有custom U-Boot layout的，我自己编译的时候看到ax6000只有stock layout和openwrt uboot，怎么才能编译出custom U-Boot layout版的

**Comment by xlighting2017** [comment]
别自己编了， Image Builder 真香

https://firmware-selector.immortalwrt.org/

<img width="1501" alt="image" src="https://github.com/immortalwrt/immortalwrt/assets/25970065/f3849c26-148d-48bd-938f-1cac2ceacb7d">

使用说明：
https://github.com/1715173329/blog/issues/9


这里面没有的，你自己编也编不出来

**Comment by 1715173329** [comment]
所谓 custom layout 已被弃用，不会再支持。

**Reply by igmkgu** [reply]
以后大分区只有itb？
